Resources/doc/reference/configuration/headers.rst

```diff
@@ -229,6 +229,22 @@ This enables a simplistic ETag calculated as md5 hash of the response body:
                     headers:
                         etag: true
 
+
+.. versionadded:: 2.2
+
+You can set up ETag to be strong or weak by passing values "strong", "weak".
```

Comment:
this section needs to be indented by 4 spaces for rst to pick it up.

lets say: ... by setting the option to "strong" or "weak" respectively.